homework9.py

```diff
@@ -0,0 +1,76 @@
+# 1. Використати файл як базу даних для збереження записів телефонної книги із попередніх завдань.
+# Ваша телефонна книга, що до цього містилася в dict, має зберігатися у вигляді тексту в JSON форматі.
+# При закритті програми і повторному відкритті всі попередні дані мають бути доступними.
+# Підказка: Ви можете використати бібліотеку json, яка допоможе із перетворенням dict в JSON string
+# (при записі в файл) і JSON string в dict (при читанні із файлу). Файл слід оновлювати після кожної
+# успішної операції add або delete.
+
+
+my_contacts = {
+    'Wednesday Addams': '+442255667788',
+    'Gregory House': '+112233445566',
+    'Winston Churchill': '+445566778899'
+}
+import json
+
+json_phone_book = json.dumps(my_contacts)
```

Comment:
I would suggest you to load the data from a file once, write it to a dict variable and work with that dict as before: you can use it without accessing the file for readonly operations (stats, list, show). And for add or delete you will need to update your dict variable and right after that dump it to a json string and rewrite the file. This will allow you to open a file only when you need to initially read or update it. Any other operation can be performed as before.

Also, I don't see here solution for 2 and 3 parts of the hometask